### DISCUSSION TITLE
Free H2 pcb/kits in North America - All gone

### DISCUSSION THREAD

**Discussion by gstwrt** [kachurovskiy/nanoels#74] [General]
I bought a set of H2 boards from jlcpcb along with components.    After assembling 2 for myself, I have 3 left over which I'm willing to send out for free within North America.

In addition to the pcb, I could send buttons, resistors, pin headers, pot, and switch.  Supply your own lcd2004 and arduino nano.

Let me know if you are interested.  I'm located in Canada btw.   Contact me at grhmstwrt _at_ gmail

**Comment by gsherwin74** [comment]
email sent
